Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,17 @@
+class Pacapt < Formula
+
+  homepage "https://github.com/icy/pacapt" 
+  url "https://github.com/icy/pacapt.git", :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4"
+  version "v2.0.2"
+
+  def install
+    system "make", "install", "BINDIR=#{bin}"
+  end
+
+  test do
+    system bin/"pacapt", "-V"
+    
+    assert_equal %x(brew list | wc -w), %x(pacapt -Q '.' | wc -w)
```

Comment:
Nice test but might be a little unreliable. Just do e.g. `system "#{bin}/pacapt", "-Q` or similar.